src/vt-tv/render/render.cc

```diff
@@ -280,35 +280,36 @@ vtkNew<vtkPolyData> Render::create_object_mesh_(PhaseWork phase) {
       }
     }
 
-    auto const& rank = this->info_.getRank(rankID);
+    auto& rank = this->info_.getRank(rankID);
 
     //@TODO: clean this stuff up
```

Comment:
Does this still need cleanup?

Reply:
Yes! I will ping you once this is sorted out 👍